src/psyclone/psyad/main.py

```diff
@@ -121,7 +120,8 @@ def msg():
     # Output the Fortran code for the adjoint kernel
     if args.oad:
         logger.info("Writing adjoint of kernel to file %s", args.oad)
-        write_unicode_file(ad_fortran_str, args.oad)
+        with open(args.oad, mode='w', encoding='utf8') as adjoint_file:
```

Comment:
Since we're here, please could you add an `encoding` to the `open` at L102?

Reply:
Done.